java/integTest/build.gradle

```diff
@@ -84,6 +85,25 @@ tasks.register('startCluster') {
     }
 }
 
+tasks.register('startClusterForAz') {
```

Comment:
Really, we can use this cluster for any test, not just az awareness. 
We should consider renaming this to something like `startClusterWithMultipleReplicaNodes` or just `startLargeCluster`

Reply:
not required for this PR, but let's consider splitting out our regular IT test framework from tests that require 'more setup'.

Reply:
Alternatively, we can move cluster setup to the IT test framework and let individual test files set up clusters